src/main/Config.cpp

```diff
@@ -129,12 +129,11 @@ Config::Config() : NODE_SEED(SecretKey::random())
     MAX_SLOTS_TO_REMEMBER = 12;
     METADATA_OUTPUT_STREAM = "";
 
-#ifdef USE_EASYLOGGING
-    LOG_FILE_PATH = "stellar-core.%datetime{%Y.%M.%d-%H:%m:%s}.log";
+#ifdef _WIN32
```

Comment:
I think we should just make the default the one with dashes or underscores and call it a day: our old default is the problem, Easylogging new this already as they use `kDefaultDateTimeFormatInFilename` as default that is `"%Y-%M-%d_%H-%m"`.

Reply:
Ok. If it breaks someone's ops logfile regex I .. guess it can be fixed there.